packages/design-system-react/src/components/button-primary/README.mdx

```diff
@@ -0,0 +1,97 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as ButtonPrimaryStories from './ButtonPrimary.stories';
+
+# ButtonPrimary
+
+ButtonPrimary is used for primary actions
```

Comment:
Description matches [insight report ](https://www.figma.com/board/SEQligIgoo7fWYGYJ9uJRJ/Button-Audit-and-Insight-Report?node-id=1214-538&t=1TyqQgoyXpuEI6Kn-1)